frontend/src/components/SideBar/SideBarLink.vue

```diff
@@ -0,0 +1,75 @@
+<script setup>
+import { computed } from 'vue';
+const props = defineProps({
```

Comment:
nitpick: please add some space around this variable declaration for better visibility